pkgs/top-level/all-packages.nix

```diff
@@ -16654,9 +16654,7 @@ with pkgs;
   svdtools = callPackage ../development/embedded/svdtools { };
 
   swiftPackages = recurseIntoAttrs (callPackage ../development/compilers/swift { });
-  inherit (swiftPackages) swift swiftpm sourcekit-lsp swift-format;
-
-  swiftpm2nix = callPackage ../development/tools/swiftpm2nix { };
```

Comment:
There should have been an addition to `aliases.nix` for this removal.

Reply:
How is it removed if this is being inherited?

![image](https://github.com/user-attachments/assets/08b1bcaf-3c7b-4c52-8658-6fc24d6f616b)

Reply:
This was not removed at all.
Indeed it was put under swiftPackages.

Reply:
> This was not removed at all. Indeed it was put under swiftPackages.

You are right :) sorry for the false alarm.